.travis.yml

```diff
@@ -21,8 +21,8 @@ before_install:
   - phantomjs --version
 
   # Use the latest stable Node.js
-  - nvm install stable
-  - nvm use stable
+  - nvm install 8.11.1
+  - nvm use 8.11.1
```

Comment:
The only reason I changed this was because there are some libs that do not support node 10 yet since it was released yesterday and yarn install was failing.

What I suggest about phantomjs is to switch to an alternative like capybara-webkit or chrome headless. Tests seem to be more flaky with these updates and it doesn't make sense. I'll see what I can do to fix this.